aws/logs_monitoring/template.yaml

```diff
@@ -154,6 +154,10 @@ Parameters:
       - true
       - false
     Description: Set to true to deploy the Forwarder to a VPC and send logs, metrics, and traces via a proxy. When set to true, must also set VPCSecurityGroupIds and VPCSubnetIds.
+  DdHttpProxyURL:
+    Type: String
+    Default: ""
+    Description: "For web proxy support when using a VPC, sets the HTTP_PROXY and HTTPS_PROXY values. Don't use this in combination with AWS Private Link. When set, must also set VPCSecurityGroupIds and VPCSubnetIds."
```

Comment:
Can you add one example? so user have higher confidence knowing what they put in is in the expected format.